windows/tools/bitmap_source_extractor_window.py

```diff
@@ -41,8 +41,8 @@ def __init__(self, app_root, *args, **kwargs):
 
         self.tags_dir = tk.StringVar(self)
         self.data_dir = tk.StringVar(self)
-        self.tags_dir.set(os.path.join(e_c.WORKING_DIR, 'tags'))
-        self.data_dir.set(os.path.join(e_c.WORKING_DIR, 'data'))
+        self.tags_dir.set(e_c.WORKING_DIR.joinpath('tags'))
+        self.data_dir.set(e_c.WORKING_DIR.joinpath('data'))
```

Comment:
Why not `Path(e_c.WORKING_DIR, "tags")` it's shorter and does the same thing.

Reply:
Because the word "joinpath" makes it more apparent what is being done without knowing how the Path constructor works. Yes, it's slightly longer, but this is the intended usage of joinpath.